Mathlib/MeasureTheory/Decomposition/SignedLebesgue.lean

```diff
@@ -189,21 +189,15 @@ variable {s t : SignedMeasure α}
 @[measurability]
 theorem measurable_rnDeriv (s : SignedMeasure α) (μ : Measure α) : Measurable (rnDeriv s μ) := by
   rw [rnDeriv_def]
-  -- Note. `measurabilty` proves this, but is very slow
-  apply Measurable.add
-  · exact ((Measure.measurable_rnDeriv _ μ).ennreal_toNNReal).coe_nnreal_real
-  · rw [measurable_neg_iff]
-    exact (Measure.measurable_rnDeriv _ μ).ennreal_toNNReal.coe_nnreal_real
+  fun_prop
 
 #align measure_theory.signed_measure.measurable_rn_deriv MeasureTheory.SignedMeasure.measurable_rnDeriv
 
 theorem integrable_rnDeriv (s : SignedMeasure α) (μ : Measure α) : Integrable (rnDeriv s μ) μ := by
   refine Integrable.sub ?_ ?_ <;>
     · constructor
-      · -- NB: `measurability` proves this, but is very slow
-        -- TODO(#13864): reinstate faster automation, e.g. by making `fun_prop` work here
-        exact (((Measure.measurable_rnDeriv _ μ
-          ).ennreal_toNNReal).coe_nnreal_real).aestronglyMeasurable
+      · apply Measurable.aestronglyMeasurable
```

Comment:
Flagging this for the future: it would be nice to avoid the `Measurable.aestronglyMeasurable` lemma; `fun_prop` should infer this instead.

Reply:
Just to be clear, in order for that to work, we need to mark `AEStronglyMeasurable` (the definition) with the `fun_prop` attribute.